rcl_logging_spdlog/include/rcl_logging_spdlog/logging_interface.h

```diff
@@ -19,6 +19,8 @@
 extern "C" {
 #endif
 
+#include <rcutils/allocator.h>
```

Comment:
A couple of nits here:

1.  It would be better to move both of the includes out of the `extern "C"` block; it really isn't necessary to have them in there.
1.  Let's use `#include "rcutils/allocator.h"`, just to be consistent with what is already there.
1.  In sort order, "rcutils/allocator.h" comes after "rcl_logging_spdlog/visibility_control.h".

Reply:
b9a2e4514598ac4f4ea5011541108f6e7b82c93c